libs missing needed in go linting

## Describe the bug

When running golangci-lint got error: "../../usr/lib/go/src/runtime/cgo/cgo.go:34:8: could not import C (cgo preprocessing failed) (typecheck)" looks like it is this one, so just need to install some libs: https://github.com/golangci/golangci-lint/issues/1176

<!-- A clear and concise description of what the bug is. -->

## Expected behavior
It do not fail
<!-- A clear and concise description of what you expected to happen. -->

## Steps to Reproduce

No idea how to reproduce this without our codebase, I can look into it if needed.

## Logs
```2021-11-16 06:50:51 [INFO]   Linting [GO] files...
2021-11-16 06:50:51 [INFO]   ----------------------------------------------
2021-11-16 06:50:51 [INFO]   ----------------------------------------------
2021-11-16 06:50:51 [INFO]   ---------------------------
2021-11-16 06:50:51 [INFO]   File:[/github/workspace/cmd/main.go]
2021-11-16 06:51:09 [ERROR]   Found errors in [golangci-lint] linter!
2021-11-16 06:51:09 [ERROR]   Error code: 1. Command output:
------
../../usr/lib/go/src/runtime/cgo/cgo.go:34:8: could not import C (cgo preprocessing failed) (typecheck)
import "C"
       ^
------
2021-11-16 06:51:09 [INFO]   ---------------------------
```
## Additional context


<!-- Add any other relevant information about the problem here. -->

--- a/Dockerfile
+++ b/Dockerfile
@@ -71,6 +71,7 @@ RUN apk add --no-cache \
     lttng-ust-dev \
     make \
     musl-dev \
+    net-snmp-dev \
     npm nodejs-current \
     openjdk11-jre \
     openssl-dev \

--- a/Dockerfile-slim
+++ b/Dockerfile-slim
@@ -70,6 +70,7 @@ RUN apk add --no-cache \
     lttng-ust-dev \
     make \
     musl-dev \
+    net-snmp-dev \
     npm nodejs-current \
     openjdk11-jre \
     openssl-dev \

